backend/server/controllers/tags.js

```diff
@@ -28,18 +28,50 @@ export const createTag = async (req, res) => {
       const existingTag = await models.tags.getTagByName(tagName)
       //Check if a tag already exists with that name
       if (existingTag !== null) {
-        const tagDTO = await TagDTO.converToDTO(existingTag)
+        const tagDTO = await TagDTO.convertToDTO(existingTag)
         res.status(200).send(tagDTO)
       } else {
         //Create a new tag and return the id
         const newTag = await models.tags.create({
           tagName: tagName,
         })
-        const tagDTO = await TagDTO.converToDTO(newTag)
+        const tagDTO = await TagDTO.convertToDTO(newTag)
         res.status(201).send(tagDTO)
       }
     }
   } catch (error) {
     res.status(500).send(error)
   }
 }
+
+//Requires Authentication
+//Response codes:
+//200 OK - successful and returns with a list of tagDTO of all tags in the database
```

Comment:
Since we do return 401 Unauthorized, should this be included in this comment? Or is it purposely left out because the future auth middleware will handle this?

Reply:
I guess I thought it could be implicitly implied by the `//Requires Authentication` but I can add it for now just so that it is more consistent and accurate, and it can be removed if we need to when the middle ware auth is handled

Reply:
I'm okay with either way